backend/README.md

```diff
@@ -2,15 +2,43 @@
 
 ## Environment setup
 
+### Pypy
+
+We **highly** recommend using Pypy.
+It makes things run almost 10x faster.
+
+```bash
+python -m virtualenv venv
```

Comment:
We excluded `venv-pypy` in `.gitignore`, but still recomment setting up pypy env under `venv`. Either remove `venv-pypy` from `.gitignore` or change `venv` -> `venv-pypy`